Integrations/MailListener-POP3/MailListener-POP3.yml

```
@@ -0,0 +1,60 @@
+commonfields:
+  id: MailListener-POP3
```

change the name, display and file names accordingly (in file names, replace space with underscore)
```suggestion
  id: Mail Listener - POP3
```